src/Standards/PEAR/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -119,33 +119,44 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
+        // Check there are no blank lines in the preamble for the property,
+        // but ignore blank lines _within_ attributes as that's not the concern of this sniff.
         if ($tokens[$commentEnd]['line'] !== ($tokens[$stackPtr]['line'] - 1)) {
             for ($i = ($commentEnd + 1); $i < $stackPtr; $i++) {
-                if ($tokens[$i]['column'] !== 1) {
+                // Skip over the contents of attributes.
+                if (isset($tokens[$i]['attribute_closer']) === true) {
+                    $i = $tokens[$i]['attribute_closer'];
                     continue;
                 }
 
-                if ($tokens[$i]['code'] === T_WHITESPACE
-                    && $tokens[$i]['line'] !== $tokens[($i + 1)]['line']
+                if ($tokens[$i]['column'] !== 1
+                    || $tokens[$i]['code'] !== T_WHITESPACE
+                    || $tokens[$i]['line'] === $tokens[($i + 1)]['line']
+                    // Do not report blank lines after a PHPCS annotation as removing the blank lines could change the meaning.
+                    || isset(Tokens::$phpcsCommentTokens[$tokens[($i - 1)]['code']]) === true
```

Comment:
Similar to the above, have you considered adding a test where `isset(Tokens::$phpcsCommentTokens[$tokens[($i - 1)]['code']]) === true` is `true`?